docs/userGuide/syntax/thumbnails.mbdf

```diff
@@ -0,0 +1,35 @@
+## Thumbnails
+
+**A `thumb` component allows you to insert thumbnails using images.**
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<thumb circle src="https://markbind.org/images/logo-lightbackground.png" size="100"/>
```

Comment:
Our examples should show good usage of the feature. The MarkBind logo after being cropped is not a good thumbnail example.